core/src/main/java/io/smallrye/stork/impl/CachingServiceDiscovery.java

```diff
@@ -42,11 +42,16 @@ public CachingServiceDiscovery(String refreshPeriod) {
         }
 
         this.lastResults = Collections.emptyList();
-        this.instances = Uni.createFrom().deferred(() -> fetchNewServiceInstances(this.lastResults)
+        Uni<List<ServiceInstance>> least = Uni.createFrom().deferred(() -> fetchNewServiceInstances(this.lastResults)
```

Comment:
`least` is not very meaningful here. I would have said something like `retrieval`.